### PR TITLE
SAK-43306 Lessons: Student without submission should appear ordered a…

### PR DESCRIPTION
**Pull request by mpellicer**
…nd similar as Assignments on Comments Tool

### CODE REVIEW

**Comment by jesusmmp** [comment]
LGTM!

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/CommentGradingPaneProducer.java [hunk 1]
```diff
@@ -204,14 +204,15 @@ public void fillComponents(UIContainer tofill, ViewParameters viewparams, Compon
 		}
 		
 		if (notSubmitted.size() > 0) {
-		    List<String> missing = new ArrayList<String>();
-		    for (String userId: notSubmitted) {
-			try {
-			    missing.add(UserDirectoryService.getUser(userId).getDisplayName());
-			} catch (Exception e) {
-			    missing.add(userId);
+			List<String> missing = new ArrayList<String>();
```
Comment (ern) [inline lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/CommentGradingPaneProducer.java]: This spacing seems off using 8 vs 4 maybe

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/QuestionGradingPaneProducer.java [hunk 2]
```diff
@@ -224,14 +224,15 @@ else if ("shortanswer".equals(questionType) &&
 		}
 		
 		if (notSubmitted.size() > 0) {
-		    List<String> missing = new ArrayList<String>();
-		    for (String userId: notSubmitted) {
-			try {
-			    missing.add(UserDirectoryService.getUser(userId).getDisplayName());
-			} catch (Exception e) {
-			    missing.add(userId);
+			List<String> missing = new ArrayList<String>();
```
Comment (ern) [inline lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/QuestionGradingPaneProducer.java]: same spacing issue

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/CommentGradingPaneProducer.java (hunk 1 above)
Comment (mpellicer) [reply lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/CommentGradingPaneProducer.java]: The spacing of that file is a mess, I think is fine, if you look before:
image

And after:
image

After the changes the spacing is consistent using tabs.

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/QuestionGradingPaneProducer.java (hunk 2 above)
Comment (mpellicer) [reply lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/QuestionGradingPaneProducer.java]: Same as above, the spacing is ok after the changes.

**Pull request merged**